### PR TITLE
EWL-6633: adds rules in order to fix search icon on IE

### PR DESCRIPTION
**Pull request by adriansmuresan**
**Jira Ticket**
- [EWL-6633: Fix search icon stretching in IE](https://issues.ama-assn.org/browse/EWL-6633)

## Description
Adjust some CSS in order to fix the stretched search icon in EI 11.

## To Test
- [ ] run `gulp serve`
- [ ] enable local SG2 testing in your D8 env
- [ ] fire up Browserstack IE11
- [ ] visit http://ama-one.local/search
- [ ] observe the search icon is no longer stretched
- [ ] Did you test in IE 11?

## Visual Regressions

_Please provide the pull request ID below. This will then reference the automated VRT report after it has been generated._

A report is available [here](http://htmlpreview.github.io/?https://github.com/AmericanMedicalAssociation/ama-style-guide-2/blob/visual-regression-testing-artifact/bugfix/EWL-6633-search-icon-in-IE/html_report/index.html).

**Before proceeding:** Run visual regression tests locally to ensure new work does not introduce new bugs.

_The visual regression tests always compare against the production version of the style guide. Changes and new work will show up as test failures. Please describe your work so that the report can be reviewed by the team._

If you are creating or updating a pattern be sure you have created or updated the [scenario in `backstop.json`](ama-style-guide-2/styleguide/backstop.json).

_After changes in the report are documented and demonstrate your desired changes, please mark the pull request as `ready for review`.

For more information on visual regression testing review the [How do I run tests?](https://issues.ama-assn.org:8446/confluence/pages/viewpage.action?pageId=23298568) document in Confluence.

## Relevant Screenshots/GIFs
- Windows 10 IE 11
image

- Windows 7 IE 11
image

## Remaining Tasks
N/A

## Additional Notes
Anything more to add? Good things to call out here are:
- are there any PRs in this or other repositories that relate to or affect this PR?
- are there any caveats or oddities testers should know about when testing this PR?
- are there any problems you ran into during your work that you'd like to call out?

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

**Comment by designerneil** [comment]
search___american_medical_association

**Comment by adriansmuresan** [comment]
@designerneil thanks for the feedback, I can't replicate that on my end. I tested in browserstack IE11 on Windows 10, 8.1 and 7.

File: styleguide/source/assets/scss/01-atoms/_search-field.scss [hunk 1]
```diff
@@ -35,9 +35,9 @@
     @extend .icon--search;
     background-color: transparent;
     background-size: 25px 30px;
-    background-position: 0 0;
-    width: 25px;
-    height: 30px;
+    background-position: center;
+    width: 25px !important;
```
Comment (designerneil) [inline styleguide/source/assets/scss/01-atoms/_search-field.scss]: I would avoid using important. Can we use specific styles to target this instead?

**Comment by adriansmuresan** [comment]
@designerneil I made some adjustments, please check this out now.

**Comment by russom-woldezghi** [comment]
@adriansmuresan I can still see the icon being a little cut-off. Can you look into this? 

screen shot 2019-02-01 at 10 17 51 am

**Comment by adriansmuresan** [comment]
@russom-woldezghi please check this out again, this should be fixed!

**Review by russom-woldezghi — APPROVED** [review]
Looks good, thanks @adriansmuresan

**Pull request merged**